recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "horovod" %}
-{% set version = "0.24.2" %}
+{% set version = "0.24.3" %}
```

Comment:
Shouldn't this be `0.25` ?

Reply:
Yes it will be 0.25. I am yet to verify 0.25 locally, once done I will push that change.